mvt/feature.go

```diff
@@ -141,56 +138,33 @@ type cursor struct {
 	x int64
 	y int64
 
-	// The dimensions for the screen tile.
-	tile tegola.BoundingBox
-
-	// The extent — it is an int, but to make computations easier and not lose precision
-	// Until we convert the ∆'s to int32.
-	extent float64
-
-	// These values are cached
-	xspan float64
-	yspan float64
+	// The tile of the screen.
+	tile *tegola.Tile
 
 	// Disabling scaling Use this when using clipping and scaling
 	DisableScaling bool
 }
 
-func NewCursor(tile tegola.BoundingBox, layerExtent int) *cursor {
-	xspan := tile.Maxx - tile.Minx
-	yspan := tile.Maxy - tile.Miny
+func NewCursor(tile *tegola.Tile) *cursor {
 	return &cursor{
-		extent: float64(layerExtent),
-		tile:   tile,
-		xspan:  xspan,
-		yspan:  yspan,
+		tile: tile,
 	}
 }
 
-//	converts a point to a screen resolution point
-func (c *cursor) ScalePoint(p tegola.Point) (nx, ny int64) {
-
-	nx = int64((p.X() - c.tile.Minx) * c.extent / c.xspan)
-	ny = int64((p.Y() - c.tile.Miny) * c.extent / c.yspan)
-
-	return nx, ny
-}
-
-func (c *cursor) MinMax() (min, max maths.Pt) {
-	return maths.Pt{X: 0 - tilebuffer, Y: 0 - tilebuffer},
-		maths.Pt{
-			X: float64(c.extent + tilebuffer),
-			Y: float64(c.extent + tilebuffer),
-		}
-}
-
+// GetDeltaPointAndUpdate assumes the Point is in WebMercator.
 func (c *cursor) GetDeltaPointAndUpdate(p tegola.Point) (dx, dy int64) {
 	var ix, iy int64
-	if c.DisableScaling {
-		ix, iy = int64(p.X()), int64(p.Y())
-	} else {
-		ix, iy = c.ScalePoint(p)
+	var tx, ty = p.X(), p.Y()
+	// TODO: gdey — We should get rid of this, as we generally disable scaling; now.
```

Comment:
any reason to keep it? If not, lets purge it now.

Reply:
It's still not sure how entangled it is. Once I've refactored the code for destructure5, I'll remove it. This is the reason for the TODO.